account_lock_date_update/README.rst

```diff
@@ -36,7 +36,7 @@ access to all technical settings.
 Usage
 =====
 
-To use this module, you need to be an account adviser and go to:
+To use this module, you need to be an Account Manager and go to:
```

Comment:
This is called Billing Administrator in 14.0: https://github.com/odoo/odoo/blob/14.0/addons/account/security/account_security.xml#L41

Reply:
Thanks, change made :+1: